sdcm/nemesis.py

```diff
@@ -2616,10 +2616,25 @@ def _mgmt_backup(self, backup_specific_tables):
 
     @latency_calculator_decorator(legend="Scylla-Manger repair")
     def disrupt_mgmt_repair_cli(self):
+        if not self.cluster.params.get('use_mgmt') and not self.cluster.params.get('use_cloud_manager'):
+            raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
+        if not self.cluster.scylla_manager_tool.feature("repair", "single-host-parallelism"):
```

Comment:
on an extra thought about that, it will affect the performance test that uses this function...
@temichus @aleksbykov @roydahan , WDYT?

Reply:
@fgelcer 
Yes, in any case, need to implement an option to explicitly set  "normal repair" for perf tests
But also we can run perf tests with single-host-parallelism and check how it affects perf tests

Reply:
yes, for perf tests we must have the ability to control it from the call, as right now we don't want to change it..
about running with this new feature, we can, and should... but it will lower the amount of work the repair is doing, so obviously it will improve the latency...

Reply:
I don't think we want this in our regular nemesis.
It will cause repairs to take very long and we don't have the issue of parallelism in our test.

Reply:
removed it from the nemesis